src/sdkRuntimeModels.ts

```diff
@@ -210,7 +210,10 @@ export type LogLevelType = 'none' | 'verbose' | 'warning' | 'error';
 // Currently, this extends MPConfiguration in @types/mparticle__web-sdk
 // and the two will be merged in once the Store module is refactored
 export interface SDKInitConfig
-    extends Omit<MPConfiguration, 'dataPlan' | 'logLevel'> {
+    extends Omit<
+        MPConfiguration,
+        'dataPlan' | 'logLevel' | 'identityCallback'
```

Comment:
when i remove this change, it still passes.  any reason for adding this?

Reply:
Weird. I probably undid the change that needed this to be added in and didn't notice. Good catch.